package.json

```diff
@@ -6,14 +6,24 @@
   "module": "dist/esm/pushin.js",
   "main": "dist/umd/pushin.js",
   "typings": "dist/esm/index.d.ts",
+  "unpkg": "dist/umd/pushin.min.js",
   "exports": {
     "./pushin.css": {
       "style": "./dist/pushin.css"
     }
   },
+  "files": [
```

Comment:
The `files` array defines what should be actually published into the registry.

You can run `npm pack` locally within the root folder, this will create the `.tar.gz` file. Then you can run:
```
$ gunzip -c pushin-4.0.0-1.tgz | tar xopf -
```
This will unzip the file into the `package` directory and you'll see what files `npm` did package for publishing.